nrf_wifi/fw_if/umac_if/inc/default/fmac_structs.h

```diff
@@ -356,6 +357,27 @@ struct nrf_wifi_fmac_priv_def {
 #endif /* CONFIG_NRF700X_STA_MODE */
 };
 
+/**
+ * @brief Structure to hold raw tx packet information.
+ *
+ * This structure holds the information sent by higher
+ * layers to transmit a raw frame.
+ */
+struct raw_tx_data {
+	/** magic number */
+	unsigned int magic_num;
```

Comment:
```suggestion
	unsigned int cookie;
```
Might be better, but I will leave it to you.

Reply:
okay

Reply:
Seems like i see magic number as a nomenclature in other code blocks elsewhere. I will keep the terminology of magic_num